### ISSUE TITLE
BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding (Delvin et al., 2019)

### ISSUE THREAD

**Issue by bebrunn** [bebrunn/Predicting-Policy-Rate-Changes-from-Central-Bank-Minutes#3]
The task for October 12th is to find a paper that instructs me to fine-tune BERT the correct way.

**Comment by bebrunn** [comment]
It seems that Delvin et al. (2019) fine-tune BERT with the following hyperparameters:

- Batch size: 16/32
- Epochs:  2, 3, 4
- Optimizer: Adam (β1 = 0.9, β2 = 0.999)
- Learning rate: 5e-5, 3e-5, 2e-5
- L2 weight decay of 0.01 with 10,000 warmup steps and linear decay
- gelu activation function
- Optional: label smooting on/off of 0.1

**Comment by bebrunn** [comment]
I need to do a search over the learning rate as suggested by Delvin et al. (2019).

**Comment by bebrunn** [comment]
Closed.
